### ISSUE TITLE
Learning rate decay is not working as expected.

### ISSUE THREAD

**Issue by mmcdermott** [mmcdermott/MEDS_EIC_AR#21]
Right now, the learning rate does not ever seem to decay.

**Comment by mmcdermott** [comment]
It seems to be the case that the default LR scheduler is being set to null for some reason.

**Comment by mmcdermott** [comment]
The raw config has the appropriate LR scheduler being set; however, the recorded hyperparameters in wandb don't show it.

**Comment by mmcdermott** [comment]
This may be because I set the monitor parameter to `tuning/loss` but it seems that what is being logged is both `tuning/loss_step` and `tuning/loss_epoch`. This may also be breaking early stopping, which likewise doesn't seem to be working.
